README.md

```diff
@@ -14,22 +14,24 @@ It follows the word's order in a sentence, which is the same as the word's colum
 <a href="https://www.buymeacoffee.com/arshadkazmi42" target="_blank"><img src="https://www.buymeacoffee.com/assets/img/custom_images/orange_img.png" alt="Buy Me A Coffee" style="height: auto !important;width: auto !important;" ></a>
 <a href="https://www.patreon.com/bePatron?u=15454240" target="_blank"><img src="https://c5.patreon.com/external/logo/become_a_patron_button.png" alt="Become a Patron!" height="40"></a>
 
-Please consider donating, if you like my work
+Please consider donating, if you like my work.
 
 ## Install
 
-```
-$ npm install ak-phrasae.js
-```
+> $ npm install ak-phrase
```

Comment:
I think, using code blocks for showing the commands would be better.
Any thoughts?

Reply:
Sure. Will update.